pegasus/helpers/section_api_helpers.rb

```diff
@@ -485,6 +485,7 @@ def script
   def to_owner_hash
     to_member_hash.merge(
       script: script,
+      course_id: @row[:course_id],
```

Comment:
Thought just to make sure I understand things - the link in your above comment is for a script cache in dashboard right? (Which doesn't help us here in pegasus). Instead we use https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/helpers/section_api_helpers.rb#L216 for our pegasus script cache.